### ISSUE TITLE
Can not update laravel-ffmpeg for Laravel 5.6

### ISSUE THREAD

**Issue by mehran-prs** [pascalbaljetmedia/laravel-ffmpeg#66]
Hi .
I want to update laravel to 5.6 from 5.5 , but getting this error when running `composer update` : 
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install symfony/process v4.0.4
    - Conclusion: don't install symfony/process v4.0.3
    - Conclusion: don't install symfony/process v4.0.2
    - Installation request for pbmedia/laravel-ffmpeg ^1.3.0 -> satisfiable by pbmedia/laravel-ffmpeg[1.3.0].
    - Conclusion: don't install symfony/process v4.0.1
    - laravel/framework v5.6.0 requires symfony/process ~4.0 -> satisfiable by symfony/process[v4.0.0, v4.0.1, v4.0.2, v4.0.3, v4.0.4].
    - laravel/framework v5.6.1 requires symfony/process ~4.0 -> satisfiable by symfony/process[v4.0.0, v4.0.1, v4.0.2, v4.0.3, v4.0.4].
    - laravel/framework v5.6.2 requires symfony/process ~4.0 -> satisfiable by symfony/process[v4.0.0, v4.0.1, v4.0.2, v4.0.3, v4.0.4].
    - Conclusion: don't install symfony/process v4.0.0
    - Installation request for laravel/framework 5.6.* -> satisfiable by laravel/framework[v5.6.0, v5.6.1, v5.6.2].
```

**Comment by pascalbaljet** [comment]
Duplicate of #65

**Comment by pascalbaljet** [comment]
Version 2.0.0 has been released with Support for Laravel 5.6!
